.circleci/config.yml

```diff
@@ -306,6 +337,11 @@ jobs:
       - run:
           name: build image
           command: |
+            export DOCKER_CLI_EXPERIMENTAL=enabled
+
+            docker run --privileged --rm tonistiigi/binfmt --install all
```

Comment:
please describe what this does, and is it possible to install only the needed platforms?

Reply:
build performed on a different processor architecture as host natively, which is linux `amd64`,  QEMU emulation need to be used to enable the build for non-native platform. qemu will modifying the host kernel, `/proc/sys/fs/binfmt_misc/`, which is why `--privileged` is required.